modules/mobile/app/controllers/mobile/application_controller.rb

```diff
@@ -42,6 +42,8 @@ def link_user_with_vets360
       unless vet360_linking_locked?(uuid)
         lock_vets360_linking(uuid)
         jid = Mobile::V0::Vet360LinkingJob.perform_async(uuid)
+        # this does come up in the logs occasionally but seems to come in batches, implying that it may be caused
+        # primarily by server outages.
         Rails.logger.info('Mobile Vet360 account link job id', { job_id: jid })
```

Comment:
Category: I am here with supporting id.
Action: leave this because there's a ticket coming up for dealing with it.